Sources/MapboxNavigation/ImageRepository.swift

```diff
@@ -1,30 +1,29 @@
 import UIKit
 
 class ImageRepository {
-    public var sessionConfiguration: URLSessionConfiguration {
-        didSet {
-            imageDownloader = ImageDownloader(sessionConfiguration: sessionConfiguration)
-        }
-    }
-
-    public static let shared = ImageRepository.init()
+    static let shared = ImageRepository()
 
     let imageCache: BimodalImageCache
-    fileprivate(set) var imageDownloader: ReentrantImageDownloader
-    private let requestTimeOut: TimeInterval = 10
```

Comment:
Doesn't we lose this timeout setting?

Reply:
I don't see why we had it in the first place, imho we can remove it.